tests/core/test_xcfunc.py

```diff
@@ -14,7 +14,6 @@ def test_xcfunc_api(self):
         ixc_1 = XcFunc.from_abinit_ixc(1)
         assert ixc_1.type == "LDA"
         assert ixc_1.name == "LDA_XC_TETER93"
-        assert ixc_1 == ixc_1
         assert ixc_1 == "LDA_XC_TETER93"
```

Comment:
i think it makes sense to keep this line but move it into a separate method `def test_eq(self): ...`

in general, the name `test_xcfunc_api` is pretty meaningless and should be split into more descriptive method names, like `test_class_methods`, `test_serialization`, or something like that

Reply:
Done in 1fea2f43f02f189b6820ab604954d975b817dfae